client/src/project/index.ts

```diff
@@ -396,10 +396,12 @@ function onEntryAddedAck(err: string, id: string) {
   }
 
   ui.entriesTreeView.clearSelection();
-  ui.entriesTreeView.addToSelection(ui.entriesTreeView.treeRoot.querySelector(`li[data-id='${id}']`) as HTMLLIElement);
+  let entry = ui.entriesTreeView.treeRoot.querySelector(`li[data-id]='${id}']`) as HTMLLIElement;

```

Comment:
(There was a typo here, the first ] shouldn't be here. I fixed it when merging.)